onnxruntime/test/shared_lib/custom_op_utils.cc

```diff
@@ -626,3 +626,28 @@ void StandaloneCustomKernel::Compute(OrtKernelContext* context) {
 
 StandaloneCustomKernel::~StandaloneCustomKernel() {
 }
+
+template<typename T>
+T MulTopCompute(const T& input_0, const T& input_1) {
```

Comment:
add to dll and extend the original case.

Reply:
+1